NEWS.md

```diff
@@ -52,6 +52,20 @@ which caused an error. We now read in the locations from a csv file.
 
 -  Added new C++ function isHeterozygous() to speed up the SIMplyBee function isCsdHeterozygous() 
 
+- parallelised all the major functions (so they run on simParamBee$nThreads)
+
+- swarm/split/supersede do no longer store the name of the queen 
+
+- colonies with high inbreeding that do not produce a viable virgin queens in 
+max(10, SP$nVirginQueens) attempts are
+removed in swarm/supersede
+
+- split no longer creates virgin queens in the split colonies but returns colonies with workers 
```

Comment:
Why? Didn't we designed the code such that both swarm and split create virgin queens, but I guess you are thinking of a beekeeper/beebreeder having control of this by providing a new queen?